src/config/config.ts

```diff
@@ -1,12 +1,21 @@
 export const INFURA_PROJECT_ID = process.env.INFURA_PROJECT_ID || "6028cd7708c54c91a90df6cefd9bf1a9"; // TODO: inject project id into env variable
-
+export const ETHERSCAN_API_KEY = (process.env.ETHERSCAN_API_KEY
+  ? JSON.parse(process.env.ETHERSCAN_API_KEY)
+  : false) || {
+  ETH: "XWKZFH99X5M1ESC6QC5YK26RRK8K9P3W9K",
+  MATIC: "RPFRR1C1XI7K2D3ZEZXT2VY9GNJPNUQ3Z3",
+};
```

Comment:
why false 🤔  we gonna fallback to the later values?

Reply:
following infura's convention, so that can fallback to default api keys